tests/target/configs/struct_field_align_threshold/20.rs

```diff
@@ -43,7 +43,7 @@ pub struct Foo {
     #[AnAttribute]
     g:     SomeOtherType,
     /// A doc comment on a field
-    h:     AThirdType,
+    h: AThirdType,
```

Comment:
When possible we want to avoid messing with unrelated tests. I think in this case we don't want to lose the alignment.

Reply:
sure